packages/tailwindcss/tests/ui.spec.ts

```diff
@@ -636,6 +624,12 @@ async function render(page: Page, content: string, extraCss: string = '') {
     @layer theme, base, components, utilities;
     @layer theme {
       ${defaultTheme}
+
+      @theme {
+        --color-red: rgb(255, 0, 0);
+        --color-green: rgb(0, 255, 0);
+        --color-blue: rgb(0, 0, 255);
+      }
```

Comment:
Added these simple colors here so we have some stable colors to use in these UI tests, otherwise these tests are going to keep getting out of sync as we refine the color palette.